doc/source/Linux.rst

```diff
@@ -0,0 +1,71 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Procedure to create symbol tables for linux
+--------------------------------------------
+
+To create symbol table please refer this :ref:`symbol-tables:Mac or Linux symbol tables`.
```

Comment:
There's a few articles missing here and there.  This would sound more natural as `To create a symbol table please`.  I'll probably do a full grammar scan of the documentation once it's all done, so don't worry about it too much at the moment.